qutip/distributions.py

```diff
@@ -357,6 +358,20 @@ def update_rho(self, rho):
 
 class HarmonicOscillatorWaveFunction(Distribution):
 
+    """
+    This class uses a function from the wavefunction_cython
+    module, which is a compiled version for the Numpy and
+    Cython version of QuTip, from the wavefunction_cython
+    module of the Fast_Wave <https://github.com/fobos123deimos/fast-wave>
+    package, which is released under
+    the BSD license, with the following copyright notice:
+    CORDEIRO, MATHEUS; BEZERRA, I. P. ; VASCONCELOS, H. H. M. . Efficient
+    Computation of the Wave Function ψn(x) using Hermite Coefficient Matrix in
+    Python. In: 7º Workshop Escola de Computação e Informação Quântica
+    (7ª WECIQ),2024, Rio de Janeiro.ANAIS DO 7º WORKSHOP-ESCOLA DE COMPUTAÇÃO
+    E INFORMAÇÃO QUÂNTICA. Rio de Janeiro: CEFET/RJ, 2024. p. 56-60.
+    """
```

Comment:
This docstring is not added to the `Distribution` one, but replace it. 
So it should have a description of what it is, it's parameters etc.

It's fine to add reference to your paper, but that not as the only description of the class.

Reply:
I solved it